content/docs/studio/user-guide/model-registry/what-is-a-model-registry.md

```diff
@@ -9,22 +9,59 @@ production, etc.) of your models on top of Git.
 https://www.youtube.com/watch?v=DYeVI-QrHGI
 
 Iterative Studio Model Registry is built on top of Git, and uses Iterative’s
-open-source Git-based tools [GTO] and [MLEM].
+open-source Git-based tools [GTO] and [MLEM] and [DVC].
+
+Model registry enables end-to-end workflows:
+
+- **Add/Log your model**: Start by logging your model's performance metrics and
+  artifacts with [dvclive]. This tool integrates seamlessly with your existing
+  ML framework, enabling you to log everything you need to later use and
+  evaluate models.
+
+- **Compare model versions using Studio**: Once you've logged your models, use
+  [Studio] UI or [DVC] CLI to compare different versions of your models. Studio
+  provides a comprehensive and intuitive interface for comparing metrics,
+  parameters, and plots across different model versions.
+
+- **Register model versions**: After comparing your models in [Studio], you can
+  register a semantic model version to mark an important iteration using
+  [Studio] UI or [GTO] CLI (Git Tag Ops). Collecting and organizing model
+  [versions] preserves their data provenance and lineage information.
+
+- **Assign stage to model**: With your model registered, you can use [Studio] UI
+  or [GTO] CLI to manage the lifecycle of your models. You can assign models to
+  specific tasks or stages (e.g. dev, shadow, prod), and promote models through
+  these stages as they prove their performance.
+
+- **Download specific model**: To use a specific model, you can download the
+  latest or requested model version, or the version in selected stage using
+  [DVC].
+
+- **Deploy with CI/CD**: To enable automation, you can setup CI/CD workflow that
+  publish or deploy your model. CI/CD will be triggered upon version
+  registration or stage assignment automatically, since they're executed by
+  creating a Git tags. To wrap your models in REST API and deploy them to
+  various platforms such as Sagemaker or Kubernetes, you can use [MLEM].
```

Comment:
I don't think we need to repeat everything from the use case page here. It takes too long to get to the subpages.